test/data.jl

```diff
@@ -231,11 +231,31 @@ end
     @test eltype(dropmissing!(copy(df), :b, disallowmissing=true).a) == Union{Int, Missing}
 
     # CategoricalArrays
-    c = Union{Int64, Missing}[1,2,1,missing]|>categorical
+    c = categorical([1, 2, 1, missing])
     df = DataFrame(c=c)
     @test dropmissing(df) == DataFrame(c=categorical([1, 2, 1]))
-    @test eltype(dropmissing(df).c) == CategoricalValue{Int64, UInt32}
-    @test eltype(dropmissing!(df).c) == CategoricalValue{Int64, UInt32}
+    @test eltype(dropmissing(df).c) == CategoricalValue{Int, UInt32}
+    @test eltype(dropmissing!(df).c) == CategoricalValue{Int, UInt32}
+
+    # Multithreaded execution test (must be at least ncol > 1, nrow > 100_000)
+    N_rows, N_cols = 10^7, 4
```

Comment:
Better not use more memory than necessary:
```suggestion
    N_rows, N_cols = 200_000_000, 4
```

Reply:
I suspect you mean `200_000`, I'll reflect the change.